### PR TITLE
Feature#27 psychologist list

### PR DESCRIPTION
**Pull request by OmarRoBra**
# Description
What did you implement/Why did you implement this:
 - List where all the psychologists will be displayed
 - Buttons to order the psychologist by alphabetical order or  rating

  
  ## Screenshots 
If applicable, include screenshots of the results or screenshots that help to see changes
image

### CODE REVIEW

File: app/views/home/psychologist_List.erb [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+          <div>
+            <% 5.times do %>
+            <%= image_tag("rateStar.png", alt: "Logo", class:"start-rate")%>
+           <% end %>
+          </div>
+        </div>
+    <% end %>
+   </div>
+  <section>
+ </div>
```
Comment (YilKanda) [inline app/views/home/psychologist_List.erb:37]: Add a line blank

**Review by YilKanda — DISMISSED** [review]
LGTM!

File: app/controllers/home_controller.rb [hunk 2]
```diff
@@ -5,7 +5,23 @@ def type;end
...
   def faqs; end
 
-  def profile_psychologist; end
+  def profile_psychologist; 
+    
+  end
+
+  def psychologist_List
+    @Psychologist = Psychologist.order(params[:value])
+    puts params[:value]
```
Comment (AlejandroVI99) [inline app/controllers/home_controller.rb:14]: If you don't need that puts you can erase it c:

File: app/controllers/home_controller.rb [hunk 3]
```diff
@@ -5,7 +5,23 @@ def type;end
...
+  end
+
+  def sort_list
+    @Psychologist = Psychologist.order(params[:value])
+    puts params[:value]
```
Comment (AlejandroVI99) [inline app/controllers/home_controller.rb:19]: Here too c:

**Review by AlejandroVI99 — DISMISSED** [review]
Nice! keep going!

File: app/assets/stylesheets/home/components/psychologist_List/psychologist_List.scss [hunk 4]
```diff
@@ -0,0 +1,94 @@
+@import "../variables";
+
+$whiteColor: #ffff;
+.List-container {
```
Comment (kevinnio) [inline app/assets/stylesheets/home/components/psychologist_List/psychologist_List.scss:4]: What do you think of using `.list-container` instead, just for consistency's sake?

File: app/assets/stylesheets/home/components/psychologist_List/psychologist_List.scss [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+  h1 {
```
Comment (kevinnio) [inline app/assets/stylesheets/home/components/psychologist_List/psychologist_List.scss:5]: You'd probably want to have an empty line between each code block.

File: app/assets/stylesheets/home/components/psychologist_List/psychologist_List.scss [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+  color: $whiteColor;
+  border: none;
+  border-radius: 15px;
+  font-weight: bold;
+  font-size: 15px;
+  padding: 4px 8px;
+  text-align: center;
+  margin: 0.5rem 0;
+}
+
```
Comment (kevinnio) [inline app/assets/stylesheets/home/components/psychologist_List/psychologist_List.scss:36]: Like this empty line for example 👍

File: app/controllers/home_controller.rb (hunk 2 above)
Comment (kevinnio) [reply app/controllers/home_controller.rb:14]: Agreed. You probably don't need it.

File: app/controllers/home_controller.rb [hunk 7]
```diff
@@ -5,7 +5,23 @@ def type;end
 
   def faqs; end
 
-  def profile_psychologist; end
+  def profile_psychologist; 
+    
+  end
+
+  def psychologist_List
```
Comment (kevinnio) [inline app/controllers/home_controller.rb:12]: In ruby, usually, method names are always lowercase. Maybe you'd like to change to this to `psychologist_list`?

File: app/controllers/home_controller.rb [hunk 8]
```diff
@@ -5,7 +5,23 @@ def type;end
...
+    @Psychologist = Psychologist.order(params[:value])
```
Comment (kevinnio) [inline app/controllers/home_controller.rb:13]: Same for instance variable names, usually all lowercase. I know is a small thing, but code style is important. :D

File: db/schema.rb [hunk 9]
```diff
@@ -11,10 +11,6 @@
 # It's strongly recommended that you check this file into your version control system.
 
 ActiveRecord::Schema.define(version: 2021_08_31_213231) do
-<<<<<<< HEAD
```
Comment (kevinnio) [inline db/schema.rb:14]: Someone must have committed a merge conflict. Thanks for fixing it!

**Review by kevinnio — CHANGES_REQUESTED** [review]
Looks good to me, but it needs small adjustments to be ready for merging.

**Review by brayandiazc — APPROVED** [review]

**Review by kevinnio — APPROVED** [review]

**Review by YilKanda — APPROVED** [review]
LGTM!

**Pull request merged**